02-express-tutorial/logger.js

```diff
@@ -0,0 +1,14 @@
+const logger = (req, res, next) => {
+  const method = req.method;
+  const url = req.url;
+  const today = new Date();
+  const date =
+    today.getFullYear() + "-" + (today.getMonth() + 1) + "-" + today.getDate();
+  const time =
+    today.getHours() + ":" + today.getMinutes() + ":" + today.getSeconds();
+  const dateTime = date + " " + time;
+
+  console.log(method, url, dateTime);
```

Comment:
## Optional Improvement
[Readability and Standardization]
Typically logs put the date time first then the "level" of the log message (debug, info, warning, error, fatal), followed by the message itself.
This makes scrolling or parsing the logs a lot easier.

So here we might see `[2024-03-30T20:14:08Z] INFO - method POST called on http://localhost:3000/api/v1/person`.

You can easily do this with string interpolation to keep is readable:
```js
const logLevel = "INFO";
const msg = `method ${method} called on ${url}`;
console.log(`[${dateTime} ${logLevel} - ${msg}`);
```